CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# v2.6.0
```

Comment:
the version should be 2.5.1 since its just a bug fix.

Reply:
later changes will be there to support API1600. Even then do I need to change version?

Reply:
i would say when that change comes, then change the version. Lets follow the semantic versioning rules for now.

Reply:
done